### PR TITLE
Fix bug in import_dir where we improperly strip off the extension.

### PR DESCRIPTION
**Pull request by andrewmchen**
Addresses #36.

### CODE REVIEW

File: tests/workspace/test_cli.py [hunk 1]
```diff
@@ -144,3 +144,25 @@ def test_import_dir_helper(tmpdir):
...
+
+def test_import_dir_rstrip(tmpdir):
+    """
+    Copy from directory ``tmpdir`` with structure as follows
+    - a (directory)
+      - test-py.py (python)
+    """
+    os.makedirs(os.path.join(tmpdir.strpath, 'a'))
+    with open(os.path.join(tmpdir.strpath, 'a', 'test-py.py'), 'wb') as f:
+        f.write('print 1 + 1')
```
Comment (loneknightpy) [inline tests/workspace/test_cli.py]: where did you check the content of this file?

Comment (andrewmchen) [reply tests/workspace/test_cli.py]: We don't check the content of the file. The test just checks to see if the paths have their extensions stripped correctly.

Comment (loneknightpy) [reply tests/workspace/test_cli.py]: If that is the case, you don't need any content here. You could just put `pass` here.

Comment (andrewmchen) [reply tests/workspace/test_cli.py]: Ok let's do that.

**Comment by loneknightpy** [comment]
lgtm

**Pull request merged**